models/_generate_layers.py

```diff
@@ -263,6 +267,14 @@ def _init_args(self):
           kind=inspect.Parameter.POSITIONAL_OR_KEYWORD),
         param_type_s="int",
         docstring="output dimension")
+    if self.layer_class in self._LayerClassesWithExplicitTarget:
+      self.params["target"] = LayerSignature.Param(
+        self,
+        inspect.Parameter(
+          name="target",
+          kind=inspect.Parameter.POSITIONAL_OR_KEYWORD),
+        param_type_s=LayerRef,
```

Comment:
This is supposed to be a string.
```suggestion
        param_type_s="LayerRef",
```